pkgs/build-support/cc-wrapper/default.nix

```
@@ -110,6 +110,9 @@ let
   gccForLibs_solib = getLib gccForLibs
     + optionalString (targetPlatform != hostPlatform) "/${targetPlatform.config}";
 
+  # Analogously to cc_solib and gccForLibs_solib
+  libcxx_solib = "${lib.getLib libcxx}/lib";
```

I would have expected to see this

```suggestion
  # Analogously to cc_solib and gccForLibs_solib
  libcxx_solib = getLib libcxx
    + optionalString (targetPlatform != hostPlatform) "/${targetPlatform.config}";
```

Why do we (or do we not) nest when cross-compiling (beside the pain it causes our tooling, apparently https://github.com/NixOS/nixpkgs/pull/279952#discussion_r1449688281).

---

Bc it was that way for darwin and i kept it, just moved to a variable